.github/workflows/build-emsdk.yml

```
@@ -85,3 +85,49 @@ jobs:
       with:
         name: pyodide-wheels
         path: wheelhouse/*.whl
+
+  build-pygbag-sdl3:
+    name: SDL3 WebAssembly build
+    runs-on: ubuntu-22.04
+    env:
+      SDK_VERSION: 3.1.61.12bi
+      SDK_ARCHIVE: python3.13-wasm-sdk-Ubuntu-22.04.tar.lz4
+      SDKROOT: /opt/python-wasm-sdk
+      SDL3_PREFIX: ${{ runner.temp }}/pygame-sdl3
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)